test/hotspot/jtreg/vmTestbase/nsk/jdi/ListeningConnector/startListening/startlis001.java

```diff
@@ -103,16 +103,13 @@ private int runIt(String argv[], PrintStream out) {
          * matches the address which was set via connector's arguments.
          * Empty host address causes listening for local connections only (loopback interface).
          * */
-        String hostname = "localhost";
         List<String> validAddresses = new LinkedList<>();
-        validAddresses.add(hostname);
+        makeValidAddresses("localhost", validAddresses);
         try {
-            Arrays.stream(InetAddress.getAllByName(hostname))
-                    .forEach(address -> validAddresses.add(address.getHostAddress()));
+            makeValidAddresses(InetAddress.getLocalHost().getHostName(),
```

Comment:
Please add a comment describing why we need to add additional addresses for InetAddress.getLocalHost().getHostName()